src/edc.c

```diff
@@ -77,6 +77,46 @@ u32 crc24q(const u8 *buf, u32 len, u32 crc)
   return crc;
 }
 
+/**
+ * Computes CRC-24Q for left-aligned bit message.
+ * This function is used for left-aligned bit messages, for example SBAS and
+ * GPS CNAV.
+ * GPS message is 300 bits total, but 276 bits without CRC. It takes 34.5
+ * 8-bit bytes, and when computing CRC the message has to be padded with zero
+ * bits.
```

Comment:
@jacobmcnamee : how did you deal with this for SBAS? Did you use the existing CRC function?